sinch/domains/voice/models/conferences/requests.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+from sinch.core.models.base_model import SinchRequestBaseModel
+
+
+@dataclass
+class GetVoiceConferenceRequest(SinchRequestBaseModel):
+    conference_id: str
+
+
+@dataclass
+class KickAllVoiceConferenceRequest(SinchRequestBaseModel):
+    conference_id: str
+
+
+@dataclass
+class ManageParticipantVoiceConferenceRequest(SinchRequestBaseModel):
+    conference_id: str
+    call_id: str
+    command: str
+    moh: str
```

Comment:
Currently, the `moh` parameter is specified as a string. While Python allows for flexibility in function parameters, I'd like to highlight the benefits of specifying the type of `moh` as `ConferenceMusicOnHold`:

1. Type Safety: Specifying the type of `moh` as `ConferenceMusicOnHold` ensures that developers use only valid values defined in the `ConferenceMusicOnHold` enum. This helps catch errors early in development and provides better type checking.
2. Self-Documenting Code: By using the enum type, we provide clear documentation for developers who will use the function in the future. It communicates the expected input and improves code readability.
3. Flexibility: Despite specifying the type, developers still have the flexibility to pass a string directly to `moh` if they prefer. For example, they can use `manage_participant(moh="ring")` instead of `manage_participant(moh=ConferenceMusicOnHold.RING.value)` if it better suits their needs.
4. Consistency: It's worth noting that we already have a [similar example](https://github.com/sinch/sinch-sdk-python/blob/82cf6b7db79a6fc508ca489dabd398124375acd9/sinch/domains/verification/models/requests.py#L9) in our codebase where enum types are used to enforce strict input validation. This change would help maintain consistency across our codebase and make it easier for developers to understand and work with our APIs.

I believe incorporating this change will enhance the clarity, robustness, and maintainability of our codebase. What are your thoughts on making this improvement?